discord.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 // VERSION of DiscordGo, follows Semantic Versioning. (http://semver.org/)
-const VERSION = "0.18.0-alpha"
+const VERSION = "0.18.0"
```

Comment:
please undo this change.

Reply:
Sorry, this change was inadvertent. Will fix.